.github/workflows/python-lint.yaml

```diff
@@ -3,27 +3,19 @@ name: Python CI
 on: [push]
 
 jobs:
-    python-black:
+    python-format:
         runs-on: ubuntu-latest
         steps:
-        - uses: actions/checkout@v3
-        - name: Python Black
-          uses: cytopia/docker-black@0.11
+        - uses: actions/checkout@v4
+        - name: Python ruff format
+          uses: astral-sh/ruff-action@v3
           with:
-            path: 'scripts/'
```

Comment:
Should this continue to specify: `src: 'scripts/'`? Or do we want to run it on everything? I'm conflicted as I don't see a lot of value in checking the Jupyter notebook below, as that is mostly just a record of my analysis.

Reply:
yea that seems fine to me. I can add that back